packages/theme/src/cli/utilities/theme-selector.ts

```diff
@@ -25,7 +25,7 @@ export async function findOrSelectTheme(session: AdminSession, options: {header?
     return filterThemes(store, themes, filter)[0]!
   }
 
-  return renderSelectPrompt({
+  return renderAutocompletePrompt({
```

Comment:
the parentMethod `findOrSelectTheme` is used in numerous commands, so this will propagate to all consumers

I don't personally see an issue with this but happy to h

Reply:
It's not an issue indeed. That's actually expected, so we may keep the DX for theme selection consistent across commands :)